### PR TITLE
Soop tmv mod

### PR DESCRIPTION
**Pull request by lbesnard**

### CODE REVIEW

**Comment by lbesnard** [comment]
blocked by https://github.com/aodn/aodn-netcdf-tools/issues/47 failing unittest

File: aodndata/templates/soop_tmv_nrt_nc_template_mooring.json [hunk 1]
```diff
@@ -93,6 +94,7 @@
       "_dimensions": ["TIME"],
       "_datatype": "float64",
       "long_name": "mass_concentration_of_inferred_chlorophyll_from_relative_fluorescence_units_in_sea_water_concentration_of_chlorophyll_in_sea_water",
```
Comment (bpasquer) [inline aodndata/templates/soop_tmv_nrt_nc_template_mooring.json]: didn't you mean "mass_concentration_of_inferred_chlorophyll_from_relative_fluorescence_units_in_sea_water" ?

File: aodndata/templates/soop_tmv_nrt_nc_template_trajectory.json [hunk 2]
```diff
@@ -93,6 +94,7 @@
       "_dimensions": ["TIME"],
       "_datatype": "float64",
       "long_name": "mass_concentration_of_inferred_chlorophyll_from_relative_fluorescence_units_in_sea_water_concentration_of_chlorophyll_in_sea_water",
```
Comment (bpasquer) [inline aodndata/templates/soop_tmv_nrt_nc_template_trajectory.json]: and here again?

File: aodndata/templates/soop_tmv_nrt_nc_template_mooring.json (hunk 1 above)
Comment (lbesnard) [reply aodndata/templates/soop_tmv_nrt_nc_template_mooring.json]: absolutely

File: aodndata/templates/soop_tmv_nrt_nc_template_trajectory.json (hunk 2 above)
Comment (lbesnard) [reply aodndata/templates/soop_tmv_nrt_nc_template_trajectory.json]: @bpasquer done !

**Pull request merged**